calendar-bundle/contao/classes/Events.php

```diff
@@ -493,6 +493,11 @@ public static function getSchemaOrgData(CalendarEventsModel $objEvent): array
 			'startDate' => $objEvent->addTime ? date('Y-m-d\TH:i:sP', $objEvent->startTime) : date('Y-m-d', $objEvent->startTime)
 		);
 
+		if ($objEvent->endDate)
```

Comment:
```suggestion
		if ($objEvent->startTime !== $objEvent->endTime)
```

Contao automatically sets the `endTime` to the `startTime` if no `endTime` or `endDate` is set.

Reply:
Oh, nice. Thank you. 👍